usaspending_api/common/helpers/data_constants.py

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
Perhaps rename to a more descriptive filename to denote these are State Name - State Code translators.

There are other functions dealing with other state data (FIPS) which could be moved to this file in the future

Reply:
I would like this file to expand beyond the scope of purely state codes and names, since we don't seem to have a library for this kind of generic real-world business logic.

Reply:
I fail to understand the vision

Reply:
It was a vague vision, but I suppose it would also be a home for things like agency codes and names (if we ever did that), maybe budget category names if we ever did that, or stuff like the words for "dollars" and "cents". Just a grab bag of stuff that is too static to justify a reference table.